mev_inspect/geth_poa_middleware.py

```diff
@@ -0,0 +1,97 @@
+from typing import (
```

Comment:
Want to call out in a docstring or comment up top that this is from web3.py until they add support?

Ideally they support this at some point and we move to that